app/src/main/java/app/abhijit/iter/LoginActivity.java

```diff
@@ -60,44 +60,44 @@
 
 public class LoginActivity extends AppCompatActivity {
 
-    private Context mContext;
-    private SharedPreferences mSharedPreferences;
-    private Cache mCache;
-    private IterApi mIterApi;
+    private Context this.Context;
+    private SharedPreferences this.SharedPreferences;
```

Comment:
Variable names must be in lowerCamelCase. Instead of `SharedPreferences`, use `sharedPreferences`.